apps/meeting/package.json

```diff
@@ -21,6 +21,7 @@
     "throttle-debounce": "^2.3.0"
   },
   "scripts": {
+    "preinstall": "node scripts/update-react-library.js",
```

Comment:
If builders without git download this repository directly from GitHub (**Download ZIP** option), they will encounter an error. The JS SDK repo actually had a few issues for this `git` dependency.

At the top of this script file, you can output an error message if `git` does not exist.